README.md

```diff
@@ -76,103 +88,156 @@ Of course, this Guide can’t replace professional advice. But talk to anyone wh
-- Options expire. You need to know how long the **exercise window** will be open.
-  - Options are only exercisable for a fixed period of time, typically seven to ten years as long as you are working for the company.
-  - ❗ Importantly, options can expire after you quit working for the company. Often the expiration is **90 days** after termination of service, making the options effectively worthless if you cannot exercise before that point.
-  - 🔹 Recently (since around 2015) a few companies are finding ways to keep the exercise window open for years after leaving a company, and promote this as fairer to employees. See [this list](https://github.com/holman/extended-exercise-windows), which includes
-[Amplitude](https://amplitude.com/blog/2015/12/01/employee-equity-is-broken-heres-our-fix/), [Clef](https://github.com/clef/handbook/blob/master/Hiring%20Documents/Guide%20to%20Your%20Equity.md), [Coinbase](https://medium.com/@barmstrong/improving-equity-compensation-at-coinbase-8749979409c3), [Pinterest](http://fortune.com/2015/03/23/pinterest-employee-taxes/), and [Quora](https://www.quora.com/Why-do-most-startups-force-employees-to-exercise-their-vested-ISO-options-within-90-days-if-they-leave-rather-than-the-option-to-convert-to-NSOs).
-- 🔸 **Vesting**: Stock and stock options may be granted to you, but they come with a variety of conditions and limitations. One of the most significant conditions is that you usually “earn” rights to the shares or options over time or under certain events. This is called **vesting**.
+[**Stock options**](https://en.wikipedia.org/wiki/Employee_stock_option) (“employee stock options,” in this case) are contracts that allow you to buy shares in the company you work for. When you buy shares, you are **exercising** the options. Options are not the same as stock; they are only the right to buy stock upon and subject to the conditions specified in the option agreement.
+
+Stock options allow you to buy shares at a *fixed* price per share, the **strike price**. The strike price is generally set lower (often much lower) than what people expect will be the future value of the stock, which means you can make money when you sell the stock.
+
+Sounds great, right? But wait just a minute. Let’s get into some of those conditions and limitations on your option agreement we mentioned above:
+
+- 🔸 **Vesting**: One of the most significant conditions is that you usually “earn” rights to the shares or options over time or under certain events. This is called **vesting**.
     - Vesting usually occurs according to a **vesting schedule**. You vest only while you work for the company. For example, it is very common to have stock or options vest over a period of four years, a bit at a time, where none of it is vested at first, and all of it is vested after four years.
```

Comment:
We use "vesting" as something that happens both to the person holding shares and to the shares themselves. Is this acceptable?

Reply:
✔️